src/lsst/cmservice/web_app/templates/pages/group_details.html

```diff
@@ -153,60 +155,85 @@
          return false
      }
     }
-    const groupScripts = {{scripts|tojson}};
-    const scriptsGridOptions = {
-    rowData: groupScripts,
-     columnDefs: [
-         {field: "name", flex: 1, cellRenderer: ScriptNameRenderer,},
-         {field: "status", flex: 1},
-         {field: "superseded", flex: 1}
-     ]
+
+    const populateScriptsGrid = (scriptsJson) => {
+        const scriptsGridOptions = {
+        rowData: scriptsJson,
+         columnDefs: [
+             {field: "name", flex: 1, cellRenderer: ScriptNameRenderer,},
+             {field: "status", flex: 1},
+             {field: "superseded", flex: 1}
+         ]
+        };
+        const scriptsGridElement = document.getElementById("scriptsGrid");
+        agGrid.createGrid(scriptsGridElement, scriptsGridOptions);
     };
-    const scriptsGridElement = document.getElementById("scriptsGrid");
-    agGrid.createGrid(scriptsGridElement, scriptsGridOptions);
+    const groupScripts = {{scripts|tojson}};
```

Comment:
This is VERY nitpicky but consider putting any rendered json in the top of the script and all together so it's clear that this is filled in by the template engine.